docs/smart-contracts/getting-started/setup.mdx

```diff
@@ -27,13 +30,29 @@ To build and develop contracts you need only a couple prerequisites:
 
 ## Install Rust
 
+### Linux, macOS, or other Unix-like OS
+
 If you use macOS, Linux, or another Unix-like OS, the simplest method to install a Rust toolchain is to install `rustup`. Install `rustup` with the following command.
 
 ```bash
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
-If you use Windows, or need an alternative method of installing Rust, check out: https://www.rust-lang.org/tools/install
+### Windows
+
+On Windows, download and run [rustup-init.exe](https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe). You can continue with the default settings by pressing Enter.
+
+:::tip
+
+The soroban CLI uses emojis. To properly render them on Windows, we recommend using [Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/). See [how to install Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/install) on Microsoft Learn.
+
+:::
+
+If you're already using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install), you can also follow the instructions for Linux.
+
+### Other
+
+For other methods of installing [Rust], check out: https://www.rust-lang.org/tools/install
```

Comment:
Small tweak to remove language we'll use elsewhere when referring to git operations.
```suggestion
For other methods of installing [Rust], see: https://www.rust-lang.org/tools/install
```